Makefile

```diff
@@ -35,13 +35,14 @@ BUILD_CONTAINER_NAME?=calico/build:latest
 ###############################################################################
 NODE_CONTAINER_DIR=calico_node
 NODE_CONTAINER_NAME?=calico/node:latest
-NODE_CONTAINER_FILES=$(shell find $(NODE_CONTAINER_DIR)/filesystem/{etc,sbin} -type f)
+NODE_CONTAINER_FILES=$(shell find $(NODE_CONTAINER_DIR)/filesystem -type f)
 # we can pass --build-arg during node image building
 NODE_CONTAINER_BUILD_ARGS?=
 NODE_CONTAINER_CREATED=$(NODE_CONTAINER_DIR)/.calico_node.created
 NODE_CONTAINER_BIN_DIR=$(NODE_CONTAINER_DIR)/filesystem/bin
-NODE_CONTAINER_BINARIES=startup allocate-ipip-addr calico-felix bird calico-bgp-daemon confd
+NODE_CONTAINER_BINARIES=startup allocate-ipip-addr calico-felix bird calico-bgp-daemon confd libnetwork-plugin
 FELIX_CONTAINER_NAME?=calico/felix:latest
+LIBNETWORK_PLUGIN_CONTAINER_NAME?=calico/libnetwork-plugin:go
```

Comment:
As discussed on Slack, let's get the libnetwork Go stuff into a `latest` build before merging this guy.